schema.ts

```diff
@@ -1245,6 +1246,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * a large blob representing the body of this entity.
```

Comment:
Nit, blob technically stands for "binary large object" but here we mean text and not binary. Would update this to be clear that this is a large text property, of rich or plain text.